### PR TITLE
Fix 274 Add pino logging to backend

### LINKED ISSUE #274: Add Pino logging to our Express web server
We've got [logging](docs/logging.md) now, so let's [add it to our Express web server](http://getpino.io/#/docs/web?id=express).

See also #204, which is doing similar work elsewhere in the backend.

### PR DESCRIPTION
**Pull request by miggs125**
Closes #274

### CODE REVIEW

**Comment by manekenpix** [comment]
Screenshot from 2019-11-18 22-52-34

No, no, abort, abort, rebase, rebase!

**Comment by miggs125** [comment]
This is what happens when I use the github gui as opposed to the command line...

**Comment by humphd** [comment]
You can fix it like this:

```
git checkout master
git pull upstream master
git checkout -B fix274 6b5b3c2ad3ca53fb620562bd648f7a05ea137189
git rebase master
...fix fix fix...
npm install
...add package-lock.json if changed...
git push origin fix274 -f
```

**Comment by humphd** [comment]
@miggs125 you're my first victim due to #295 landing!  Can you please rebase on master, and get rid of the changes to package-lock.json.

If it were me, I might do something like this:

```
git checkout fix274
git checkout -- package*
git checkout master
git pull upstream master
git checkout -
git rebase master
npm install --save express-pino-logger pino-pretty
git add package.json
git commit ...
```

If it goes wrong, I'll help fix it.

**Comment by miggs125** [comment]
@humphd Thanks for the help!

**Comment by humphd** [comment]
How painful was that?

Also, I notice your earlier PR set had a version bump on `pino-pretty` to `3.4.0`, in case you want to do that in here.

**Comment by miggs125** [comment]
@humphd I actually ran into the error before I could read your code but I figured it out. Wasn't super painful but it took a couple of tries

**Comment by humphd** [comment]
Flagging @hansal7014 in on this review, since he's worked on the logging code in the past.

File: src/count-blog-domain.js [hunk 1]
```diff
@@ -7,9 +7,9 @@
 const url = require('url');
 
 //  import logger
-const parentLogger = require('./lib/logger');
+const { logger } = require('./lib/logger');
 
-const log = parentLogger.child({ module: 'count-blog-domain' });
+const log = logger.child({ module: 'count-blog-domain' });
```
Comment (humphd) [inline src/count-blog-domain.js]: One thing we'll have to do in another PR is get better at naming our "modules" for the logger.  Each class doesn't need its own module name.  Rather, we should lump them together into logical components in the app.

File: src/lib/logger.js [hunk 2]
```diff
@@ -11,17 +13,37 @@ let logger;
  */
 if (process.env.NODE_ENV === 'development') {
   logger = pino({
-    prettyPrint: { translateTime: 'SYS: yyyy-mm-dd HH:MM:ss.l ' },
+    prettyPrint: {
+      translateTime: 'SYS: yyyy-mm-dd HH:MM:ss.l ',
+      colorize: !(os.type() === 'Windows_NT'),
```
Comment (humphd) [inline src/lib/logger.js]: Nice catch on Windows and colour issues.

**Comment by hansal7014** [comment]
Changes look good to merge. Nice work @miggs125.
